account_central_billing/tests/test_res_partner.py

```diff
@@ -78,14 +75,12 @@ def test_get_invoicing_partner_vals(self):
         self.assertEqual(
             self.hq.get_billing_partner(vals),
             self.company.partner_id,
-            "Centrally billed partners of subsidiary should invoice "
-            "main company",
+            "Centrally billed partners of subsidiary should invoice " "main company",
```

Comment:
All of this file, remove the excess quotes in the strings that have pulled up a line.